tutorials/terraform/data-transfer/data-transfer-mkf-mkf.tf

```diff
@@ -0,0 +1,157 @@
+# Infrastructure for the Yandex Cloud Managed Service for Apache Kafka®, and Data Transfer.
```

Comment:
Infrastructure for the Yandex Cloud Managed Service for Apache Kafka® and Data Transfer.